dispatcher/config.ini

```diff
@@ -36,6 +36,13 @@ RetryReset = 3
 # Min time between Stop-Arm and Arm-Start commands
 TimeBetweenCommands = 6
 
+# Time between reader and CC start
+# THIS IS AN IMPORTANT VALUE, DON'T CHANGE IT UNLESS YOU KNOW WHAT YOU'RE DOING
+# 20210308 - Darryl
+StartCmdDelay = 1
```

Comment:
Why the change from 1.5 to 1?

Reply:
1 was the intended value originally. I changed it to 1.5 after The Incident to be extra safe while we dug into what happened. In principle it can be as low as 105ms.